modules/tinymce/src/core/test/ts/browser/content/EditorNbspTest.ts

```diff
@@ -0,0 +1,35 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('tinymce.src.core.test.ts.browser.content.EditorNbspTest', () => {
```

Comment:
```suggestion
describe('browser.tinymce.core.content.EditorNbspTest', () => {
```
This is named incorrectly.

Reply:
Also these tests should be in the `SpaceKeyTest` instead, as that's where all the existing tests are.

- [ ] Move the tests to the `SpaceKeyTest.ts` file